admin/app/jobs/RefreshFrontsJob.scala

```diff
@@ -63,8 +54,7 @@ object RefreshFrontsJob extends Logging {
   def runAll(): Option[Seq[Unit]] = {
     Configuration.aws.frontPressSns.map(Function.const {
       log.info("Putting press jobs on Facia Cron (MANUAL REQUEST)")
-
-      for {update <- getAllCronUpdates.getOrElse(Nil)}
+      for {update <- getAllCronUpdates}
```

Comment:
In scala I tend to go for the policy of least amount of changes possible 👅  But I've replaced curly with round.